backend/src/main/java/com/onair/hearit/common/exception/ErrorCode.java

```diff
@@ -8,12 +8,18 @@ public enum ErrorCode {
 
     //CLIENT ERROR
     INVALID_INPUT(HttpStatus.BAD_REQUEST, "입력값이 유효하지 않습니다."),
-    UNAUTHORIZED(HttpStatus.UNAUTHORIZED, "인증되지 않은 요청입니다."),
     NOT_FOUND(HttpStatus.NOT_FOUND, "해당 정보를 찾을 수 없습니다."),
     METHOD_NOT_ALLOWED(HttpStatus.METHOD_NOT_ALLOWED, "지원하지 않는 HTTP 메서드입니다."),
     ALREADY_EXIST(HttpStatus.CONFLICT, "이미 존재하는 데이터입니다."),
     BUFFER_OVERFLOW(HttpStatus.TOO_MANY_REQUESTS, "대기 중인 요청이 너무 많습니다."),
 
+    //CLIENT UNAUTHORIZED ERROR
+    UNAUTHORIZED(HttpStatus.UNAUTHORIZED, "인증되지 않은 요청입니다."),
+    UNAUTHENTICATED(HttpStatus.UNAUTHORIZED, "엑세스 토큰이 존재하지 않습니다."),
```

Comment:
비회원이 북마크, 라이브러리 등을 접근한거니까
```suggestion
AUTHENTICATION_REQUIRED(HttpStatus.UNAUTHORIZED, "로그인이 필요한 기능입니다."),
```
어떤가요

Reply:
반영했습니다! 

[feat: 비회원용 인증필요 ErrorCode 이름 명확하게 변경](https://github.com/woowacourse-teams/2025-hEARit/pull/554/commits/be86912f2963ee7f4f4d13b47dd18cfe2b1052e5) https://github.com/woowacourse-teams/2025-hEARit/issues/549